framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/resource/OpenShiftResourceConstants.java

```diff
@@ -30,7 +30,6 @@
     public static final String DEPLOYMENT_CONFIG_LABEL = "deploymentConfig";
 
     // Route constants
-    public static final String CENTRAL_CI_ROUTE_SUFFIX = ".project.openshiftdomain";
```

Comment:
Good catch. This Central CI dependent constant should not be present in this repo.